system/disk_latency.tf

```diff
@@ -0,0 +1,45 @@
+resource "datadog_monitor" "disk_read" {
+  name    = "Disk read time overloaded ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "Disk read time overloaded ${var.read_percent_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
```

Comment:
`data.aws_instance.monitored.instance_id.public_ip` should be `data.aws_instance.monitored.public_ip`

Reply:
Check the other files too.